monkey/common/agent_configuration/agent_sub_configurations.py

```diff
@@ -79,15 +79,16 @@ class ScanTargetConfiguration(MutableInfectionMonkeyBaseModel):
                             Example: ("1.1.1.1", "2.2.2.2")
         :param inaccessible_subnets: Subnet ranges that shouldn't be accessible for the agent
                                      Example: ("1.1.1.1", "2.2.2.2/24", "myserver")
-        :param local_network_scan: Whether or not the agent should scan the local network
+        :param scan_local_interfaces: Whether or not the agent should scan the machine's
+            network interfaces in addition to the provided subnet ranges
```

Comment:
We need to avoid the word "local". It implies that only local area networks (LANs) will be scanned.